odmObject/odmobject-ipso-humidity.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
can we remove this file from this pull request and add the changes to the OCF humidity?